SWLOR.Game.Server/Feature/ShipModuleDefinition/CombatLaserModuleDefinition.cs

```diff
@@ -99,5 +103,31 @@ public class CombatLaserModuleDefinition: IShipModuleListDefinition
                 });
         }
 
+        private static void PlayShipHitSfx(uint target, int amount)
+        {
+            if (amount < 0) return;
+
+            var targetShipStatus = Space.GetShipStatus(target);
+
+            var remainingDamage = amount;
+            // First deal damage to target's shields.
+            if (remainingDamage <= targetShipStatus.Shield)
+            {
+                // Shields have enough to cover the attack.
+                targetShipStatus.Shield -= remainingDamage;
```

Comment:
Space.ApplyShipDamage on line 84 is already doing this. This will apply damage twice. This section should be removed. 

I might be ok with those two VFX going into Space.ApplyShipDamage if you want to keep it though.